src/sonic-yang-models/yang-models/sonic-lldp.yang

```diff
@@ -0,0 +1,155 @@
+module sonic-lldp {
+    namespace "http://github.com/Azure/sonic-lldp";
+    prefix slldp;
+    yang-version 1.1;
+
+    import sonic-port {
+        prefix prt;
+    }
+
+    import sonic-extension {
+        prefix sonic-ext;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONiC LLDP yang model";
+
+    revision 2021-07-08 {
+        description
+            "Initial revision.";
+    }
+
+    grouping lldp_mode_config {
```

Comment:
Any reason by grouping is used?

Reply:
Enabled and mode leaf is used in Global and per Port config. So it was grouped and used in these nodes. However usage of grouping is leading to compilation error, its commented for now. But ideally since its used in two places, grouping is defined.